integreatly/integreatly-1.7.6/installation.crd.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Does this need to be committed? I don't see an `integreatly` manifest on the master branch, so I'm assuming it's kept in the operator repo

Reply:
Just checked with Jameel, she confirmed

Reply:
A lot of changes in 3 day :+1: